### PR TITLE
Mobile Response for PageTitle

### PR DESCRIPTION
**Pull request by hdave07**
Mobile Screen:
image
Ipad Screen: 
image
Computer:
image

### CODE REVIEW

File: src/components/PageTitle.jsx [hunk 1]
```diff
@@ -3,19 +3,19 @@ import wave from "@/public/waveBottom.webp";
...
       />
-      <p className="flex items-center justify-center absolute top-0 font-bold text-5xl text-white border-swim-yellow border-r-[20px] w-1/3 h-full bg-swim-blue-200/70">
+      <p className="flex items-center justify-center absolute top-0 font-bold text-2xl md:text-5xl text-white border-swim-yellow border-r-[20px] w-1/3 h-full bg-swim-blue-200/70">
         {title}
       </p>
       <Image
         src={wave}
         alt="wave bottom image"
-        className="absolute bottom-0 transform -scale-x-100 w-screen"
+        className="absolute bottom-0 transform -scale-x-100 w-screen md:w-auto "
```
Comment (seanquiambao) [inline src/components/PageTitle.jsx:18]: image

the wave is broken on desktop view

**Review by seanquiambao — CHANGES_REQUESTED** [review]
great job so far! i think the wave is the only thing you need to fix.

**Comment by seanquiambao** [comment]
link issue

**Comment by hdave07** [comment]
With *80 Zoom:
image

**Comment by hdave07** [comment]
https://github.com/acm-ucr/ucr-swim-website/issues/75

File: src/components/PageTitle.jsx [hunk 2]
```diff
@@ -3,19 +3,19 @@ import wave from "@/public/waveBottom.webp";
...
       />
-      <p className="flex items-center justify-center absolute top-0 font-bold text-5xl text-white border-swim-yellow border-r-[20px] w-1/3 h-full bg-swim-blue-200/70">
+      <p className="flex items-center justify-center absolute top-0 font-bold text-2xl md:text-5xl text-white border-swim-yellow border-r-[20px] w-1/3 h-full bg-swim-blue-200/70">
         {title}
       </p>
       <Image
         src={wave}
         alt="wave bottom image"
-        className="absolute bottom-0 transform -scale-x-100 w-screen"
+        className="absolute bottom-0 transform -scale-x-100 w-screen md:w-auto lg:w-screen"
```
Comment (seanquiambao) [inline src/components/PageTitle.jsx:18]:
```suggestion
        className="absolute bottom-0 transform -scale-x-100 w-screen"
```

can you do this instead?

**Review by seanquiambao — CHANGES_REQUESTED** [review]

**Comment by hdave07** [comment]
Yeah, it does not cause a problem. It works for all, the bigger and smaller screens.

**Pull request merged**